src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -789,6 +796,26 @@ def pickRSE(self, rseExpression='rse_type=TAPE\cms_type=test', rseAttribute='ddm
         choice = weightedChoice(rsesWithWeights)
         return choice
 
+    def requiresApproval(self, rse):
+        """
+        _requiresApproval_
+
+        Returns wether or not the specified RSE requires rule approval.
+        :param res: Rucio RSE name
+        :returns: True if the RSE requires approval to write (rule property)
+        """
+        
```

Comment:
German, let me suggest to remove this blank line. According to the PEP convention, there shouldn't be blank lines after the docstring (and I agree that its absence makes the code more readable). If you want, see:
https://www.python.org/dev/peps/pep-0257/